### DISCUSSION TITLE
SimpleMLP Dropout probability

### DISCUSSION THREAD

**Discussion by zafstojano** [ContinualAI/avalanche#898] [Feature Request] [answered]
Dear Avalanche community,

I believe I have found a tiny bug in the `SimpleMLP` class.

In particular, the `drop_rate` from the `__init__` method is not passed to `nn.Dropout` for the hidden layers of the MLP.
In turn, this instantiates a Dropout layer with default probability `p=0.5`.

I have created a simple pull request solving this issue.

Please let me know if I should perform anything differently.

Best regards,
Zafir

**Comment by vlomonaco** [comment, marked as answer]
Thanks @zaf-stojano! It was merged in #896! :)

**Reply by zafstojano** [reply]
Cheers 🍻
